test/district_repository_test.rb

```diff
@@ -8,87 +8,88 @@ def fixture_path
     File.expand_path("fixtures/Kindergartners in full-day program.csv", __dir__)
   end
 
-  def dr
-    @dr ||= begin
-      dr = DistrictRepository.new
-      dr.load_data({
-        :enrollment => {
-          :kindergarten => fixture_path
-        }
-       })
-       dr
+  def setup
```

Comment:
Having setup/teardown methods in our test harnesses is necessary to show that our tests can work in isolation of other tests and aren't affected by any asynchronous properties of our test suite.